ignition/service/framework.py

```diff
@@ -193,7 +196,7 @@ def __register_service_capabilities(self, service_class):
 
     def __register_service_requirements(self, service_class, required_capabilities):
         for requirement_name, required_capability_class in required_capabilities.items():
-            if not issubclass(required_capability_class, Capability):
+            if required_capability_class is not None and not issubclass(required_capability_class, Capability):
```

Comment:
When would the required_capability_class be None?

Reply:
Added in error, will remove.